packages/client/docusaurus/docs/client/05-advanced/03-permissions-and-capabilities.mdx

```diff
@@ -3,4 +3,147 @@ id: 03-permissions-and-capabilities
 title: Permissions and Capabilities
 ---
 
-Stay tuned for more information on this topic.
+import SDKSpecific from '../SDKSpecific';
+
+## Introduction
+
+In some types of calls, there is a common requirement for participant moderation.
+A typical example is a webinar where the host wants to control who can speak or who can share their video or screen.
+
+Stream Video SDK provides a certain set of permissions and capabilities
+that can be used to control the behavior of participants in a call.
+
+## Permissions
+
+As soon as you join a call, the `Call` instance would allow you to check the permissions of the local user
+or perform some permission-related actions:
+
+### Check permissions
+
+```ts
+import { OwnCapability } from '@stream-io/video-client';
+
+const call = streamVideoClient.call(type, id);
+const canSendAudio = call.permissionsContext.hasPermission(
+  OwnCapability.SEND_AUDIO,
+);
+```
+
+<SDKSpecific name="react">
+In our React Video SDK, you can use the [`useHasPermissions`](../04-call-engine/hooks.md#useHasPermissions) hook to check for permissions.
+
+```tsx
+const canSendAudio = useHasPermissions(OwnCapability.SEND_AUDIO);
+```
+
+</SDKSpecific>
+
+### Request permissions
+
+Every user may request permissions to perform certain action depending on the call type and call settings.
+For example, in an audio-room call type, only the hosts have `send-audio` permission by default.
+Other users should request this permission before they can start sending audio if the call settings allow it.
+
```

Comment:
Could we provide an overview over the different call types and the default permissions here? 

Not sure it's the best place, but that could be incredibly helpful to have a place to go to to see that...

I'm thinking of a simple table maybe...

Reply:
I think we should/would have a special chapter for the Call Types overview, together with their default configuration, roles, permissions, and capabilities. Ideally, this page would be shared across all SDKs as the content of it would be generic.

Notion reference:
- https://www.notion.so/stream-wiki/Call-Types-Permissions-9741753a0d77428fb4ff51dc1328314e

Reply:
I've created a task for you on our Notion board.